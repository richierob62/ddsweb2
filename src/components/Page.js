import React from 'react';

const Page = (props) => {
    
    return (
        <div className="page">
            {props.children}
        </div>
    );
}

export default Page;