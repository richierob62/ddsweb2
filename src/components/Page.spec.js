import React from 'react';
import Page from './Page';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';


it('renders without crashing', () => {
  const el = shallow(<Page />);
  expect(shallowToJson(el)).toMatchSnapshot();
});
