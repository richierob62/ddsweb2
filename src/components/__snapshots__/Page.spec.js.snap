exports[`test renders without crashing 1`] = `<styled.div />`;
